pom.xml

```diff
@@ -8,6 +8,10 @@
         <version>1</version>
     </parent>
 
+    <prerequisites>
```

Comment:
Sorry, dette er ikke for å enforce Maven-versjon som man bruker for bygg, og man får warn om det i bygget:
> [WARNING] The project no.digipost:certificate-validator:jar:1.5-SNAPSHOT uses prerequisites which is only intended for maven-plugin projects but not for non maven-plugin projects. For such purposes you should use the maven-enforcer-plugin. See https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html

Reply:
Stusset litt på det der, _enforcer-plugin_ sier:
```
<requireMavenVersion>
     <version>3.0.3</version>
<requireMavenVersion>
```,
men likevel klager prosjektet på at det ikke er satt eksplisitt i prosjektet. Hvorfor er det slik?

Reply:
I Signering er følgende under plugin:
```
<plugin>
    <groupId>org.jasig.maven</groupId>
    <artifactId>maven-notice-plugin</artifactId>
    <executions>
        <execution>
            <id>check-NOTICE-file</id>
            <phase>verify</phase>
            <goals>
                <goal>check</goal>
            </goals>
        </execution>
    </executions>
</plugin>
```
, men i sertifikatvalidatoren:
```
<plugin>
<artifactId>maven-enforcer-plugin</artifactId>
<executions>
    <execution>
        <id>test</id>
        <goals>
            <goal>enforce</goal>
        </goals>
        <phase>test</phase>
    </execution>
</executions>
</plugin>
```

Jeg tror vi må ta en prat om hvor forskjellige disse er og hvorfor det er slik. Og hvorfor `<PluginManagement>` har en whitelist i dette repoet, mens det er en blacklist i Signering på hvilke avhengigheter man sjekker.

Reply:
Fra docs: `includes - a list of artifacts to include. These are exceptions to the excludes.` Bare jeg som synes dette er litt forvirrende? 😏

Reply:
Både `digipost`, `signering` og `certificate-validator` har alle warns om at versjon ikke er satt spesifikt. `signature-api-client-java` arver en mye nyere versjon, men jeg skjønner ikke hvor den kommer fra.

Reply:
💘[https://github.com/mojohaus/versions-maven-plugin/issues/195](Rune Rune Rune) 💘